src/models/DataSets.js

```diff
@@ -27,10 +29,33 @@ export async function get(d2, filters, pagination) {
     // order=FIELD:DIRECTION where direction = "iasc" | "idesc" (insensitive ASC, DESC)
     const [field, direction] = sorting || [];
     const order = field && direction ? `${field}:i${direction}` : undefined;
+    const vaccinationIds = await getByAttribute(d2);
     const filter = _.compact([
         searchValue ? `displayName:ilike:${searchValue}` : null,
         showOnlyUserCampaigns ? `user.id:eq:${d2.currentUser.id}` : null,
+        `id:in:[${vaccinationIds.join(",")}]`,
     ]);
     const listOptions = { fields, filter, page, pageSize, order };
     return d2.models.dataSets.list(cleanOptions(listOptions));
 }
+
+async function getByAttribute(d2) {
+    const appCode = metadataConfig.attibuteCodeForApp;
+    const filter = `attributeValues.attribute.code:eq:${appCode}`;
+    const listOptions = {
+        filter,
+        fields: ["id", "attributeValues[value, attribute[code]]"],
+        paging: false,
+    };
+    const dataSets = await d2.models.dataSets.list(listOptions);
+    const ids = dataSets
+        .toArray()
+        .filter(
+            ({ attributeValues }) =>
+                !!attributeValues.find(
```

Comment:
Equivalent and more descriptive: `attributeValues.some`

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some